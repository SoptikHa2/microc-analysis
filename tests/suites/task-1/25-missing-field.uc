 main() {
   var r;
   r = { x: 1 };
   r.y = 42;
   return r.x;
 }
