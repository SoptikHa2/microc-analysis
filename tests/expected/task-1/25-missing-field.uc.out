---
test: run suites/task-1/25-missing-field.uc
---
suites/task-1/25-missing-field.uc:4:5: execution error: Non-existing field (y) access in {x:1}
     r.y = 42;
      ^


exit: 1
